modules/map/tools/map_datachecker/server/alignment_agent.hpp

```diff
@@ -0,0 +1,195 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+#ifndef _MODULES_MAP_TOOLS_MAP_DATACHECKER_ALIGNMENT_AGENT_HPP
+#define _MODULES_MAP_TOOLS_MAP_DATACHECKER_ALIGNMENT_AGENT_HPP
+#include <grpc++/grpc++.h>
+#include <chrono>
+#include <thread>
+#include <memory>
+#include <vector>
+#include "modules/map/tools/map_datachecker/server/static_align.h"
```

Comment:
add the libs in their names' alphabet order

Reply:
OK. But in some cases, this rule may be problematic.